shell/src/main/java/org/apache/accumulo/shell/Shell.java

```diff
@@ -812,23 +808,23 @@ public void execCommand(String input, boolean ignoreAuthTimeout, boolean echoPro
   private ShellCompletor setupCompletion() {
     rootToken = new Token();
 
-    Set<String> tableNames = null;
+    Set<String> tableNames;
     try {
       tableNames = accumuloClient.tableOperations().list();
     } catch (Exception e) {
       log.debug("Unable to obtain list of tables", e);
       tableNames = Collections.emptySet();
     }
 
-    Set<String> userlist = null;
+    Set<String> userlist;
```

Comment:
IIRC we had to initialize variables to null, and now we don't?

Reply:
I believe the ide is now smart enough to see that the value is set on all paths before it is used, so the initial set is redundant.  The IDE flags it with 
```
Variable 'userlist' initializer 'null' is redundant
```

On the other hand, this will be flagged as an error - `variable n might not have been initialized`

```
   String n;
    if(n == null){

    }
```